src/traces/scattermapbox/convert.js

```diff
@@ -184,7 +184,7 @@ function makeCircleGeoJSON(calcTrace, hash) {
     // Translate vals in trace arrayOk containers
     // into a val-to-index hash object
     function translate(props, key, val, index) {
-        if(!hash[key][val]) hash[key][val] = index;
+        if(hash[key][val] === undefined) hash[key][val] = index;
```

Comment:
Here `hash[key][val]` store indices in of `arrayOk` attribute. This check :arrow_double_up:  was _failing_ if a repeated value for first found in the first `marker.size` or `marker.color` item as `0` is falsy.